src/main/java/InputData/editPlaylistInputData.java

```diff
@@ -28,7 +28,7 @@ public Song getSong(){
         return this.song;
     }
     public String getNewName(){
-        return newName;
+        return this.newName;
```

Comment:
Thanks for catching this part in my editPlaylist and newPlaylist input data classes, I missed this